azure-resources/Sql/servers/recommendations.yaml

```diff
@@ -6,9 +6,9 @@
   recommendationResourceType: Microsoft.Sql/servers
   recommendationMetadataState: Active
   longDescription: |
-    If your primary database fails, perform a manual failover to the secondary database which remains read-only until then. Active geo-replication allows creating readable replicas and manual failover in case of a datacenter outage or application upgrade.
+    Active Geo Replication is a business continuity solution that utilizes a readable secondary database replica. In the event of a primary database failure, perform a manual failover to the secondary database and ensure your application points to the appropriate endpoint. Until you fail over, the secondary database remains read-only. Active geo-replication enables you to create readable replicas and manually failover to any replica if there is a datacenter outage. Up to four secondaries are supported in the same or different regions, and the secondaries can also be used for read-only access queries.
```

Comment:
Please re-word to bring the longDescription under the 300 character limit, including spaces.

Reply:
Fixed